RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 -----
 * [**] Fix editor crash occurring on large posts [https://github.com/wordpress-mobile/WordPress-Android/pull/20046]
 * [*] [Jetpack-only] Site Monitoring: Add Metrics, PHP Logs, and Web Server Logs under Site Monitoring [https://github.com/wordpress-mobile/WordPress-Android/issues/20067]
+* [**] Prevent images from temporarily disappearing when uploading media [https://github.com/WordPress/gutenberg/pull/57869]
```

Comment:
@derekblank We usually put the prefix `Block Editor: ` in the release notes for entries related to the editor.